docs/Manual/03 Schedules.md

```diff
@@ -64,7 +64,7 @@ for k in range(11):
 ```
 
 However, some orders are not allowed. Describing these restrictions in full will require concepts that are yet to be introduced. Therefore, we are stating these restrictions here and will discuss them later in the upcoming sections. The restrictions are: 
-1. The *inner dimension* created by a `split` transformation (see below) is always followed by its corresponding *outer dimension*.
+1. The *inner dimension* created by a split transformation (see below) must be ordered after its corresponding *outer dimension*.
```

Comment:
oops, needs to be `split` (with the backticks)

Reply:
Noted. Sorry, my bad.